lib/rubocop/cop/chef/modernize/respond_to_provides.rb

```diff
@@ -18,19 +18,20 @@ module RuboCop
   module Cop
     module Chef
       module ChefModernize
-        # It is not longer necessary respond_to?(:foo) in metadata. This was used to support new metadata
-        # methods in Chef 11 and early versions of Chef 12.
+        # In Chef Infra Client 12+ is is no longer necessary to gate the use of the provides methods in resouces by appending either `if respond_to?(:provides)` or `if defined? provides`.
```

Comment:
```suggestion
        # In Chef Infra Client 12+ is is no longer necessary to gate the use of the provides methods in resouces with `if respond_to?(:provides)` or `if defined? provides`.
```

Maybe reads better simplified?